### PR TITLE
Reframe computational screens and tie limitations together across sections

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

I want to give this one more read-through in the morning (and fix the merge conflict), but this should be the substantive changes. I'm putting it up because I know it took me much longer than promised!

### Suggested reviewers (optional)

@agitter @SiminaB

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/20.pharmaceuticals.md [hunk 1]
```diff
@@ -435,100 +435,136 @@ Thus, advances in biologics for COVID-19 illustrate that an understanding of how
...
+Computational analysis correctly identified the Sigma-1 receptor as a protein of interest, but the process of identifying which drugs might modulate the interaction led to an emphasis on candidates that ultimately have not been supported. 
+In comparison, computational techniques were used to design compounds targeting M^Pro^ based on an understanding of how this protease aids viral replication, and M^Pro^ inhibitors remain promising candidates [@doi:10.1039/D0CS01065K], although the clinical trial data is not yet available.
+Despite the concerns it raises about high-throughput screening, it was a data-driven approach that allowed researchers to notice a shared property among many of the hits from computational screening analyses [@doi:10.1126/science.abi4708].
+In combination, these findings suggest that data-driven research can be an important component of the drug repurposing ecosystem, but that drug repurposing efforts that proceed without a hypothesis or without an emphasis on biological mechanisms are not likely to produce valuable candidates.
```
Comment (cgreene) [inline content/20.pharmaceuticals.md:532]: This is pretty strong ("are not likely"). The evidence looks that way... but perhaps different screens in the future will be more valuable? I wonder if there's a way to lean on the sentence that follows. Like, what if it ended with "may not produce valuable candidates."

Comment (rando2) [reply content/20.pharmaceuticals.md:532]:
```suggestion
In combination, these findings suggest that data-driven research can be an important component of the drug repurposing ecosystem, but that drug repurposing efforts that proceed without a hypothesis or without an emphasis on biological mechanisms may not produce valuable candidates.
```
Thanks @cgreene!

**Review by cgreene — APPROVED** [review]

**Comment by rando2** [comment]
With Casey's review, I'm planning to merge this in within the next couple hours and email out a track-changes version of the manuscript as it stands to all the coauthors. If anyone wants to make suggestions on this text after that, you are definitely welcome to either open a PR or get in touch about the best way to get your thoughts integrated!

**Comment by agitter** [comment]
@rando2 that works for me.  I plan to read these new changes more carefully.  I won't get to it this afternoon, so I can do that post-merge.

**Comment by rando2** [comment]
That sounds perfect, thank you @agitter!

**Pull request merged**